{"metadata":{"modules":[{"name":"IdVerification365id"}],"externalID":"c:@M@IdVerification365id@E@IdVerificationError@IdVerificationErrorServerError","symbolKind":"case","title":"IdVerificationError.serverError","roleHeading":"Case","role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"serverError","kind":"identifier"}]},"sections":[],"identifier":{"url":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/IdVerificationError\/serverError","interfaceLanguage":"swift"},"primaryContentSections":[{"declarations":[{"platforms":["iOS"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"serverError","kind":"identifier"}],"languages":["swift"]}],"kind":"declarations"}],"kind":"symbol","abstract":[{"text":"Happens when there is an error thrown by the 365id server","type":"text"}],"hierarchy":{"paths":[["doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id","doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/IdVerificationError"]]},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/idverification365id\/idverificationerror\/servererror"]}],"schemaVersion":{"major":0,"patch":0,"minor":3},"references":{"doc://com.365id.sdk.idverification/documentation/IdVerification365id":{"abstract":[],"identifier":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id","type":"topic","url":"\/documentation\/idverification365id","title":"IdVerification365id","role":"collection","kind":"symbol"},"doc://com.365id.sdk.idverification/documentation/IdVerification365id/IdVerificationError":{"navigatorTitle":[{"kind":"identifier","text":"IdVerificationError"}],"fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"IdVerificationError"}],"role":"symbol","kind":"symbol","identifier":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/IdVerificationError","abstract":[{"type":"text","text":"Id Verification Errors used in the "},{"code":"onError(error:)","type":"codeVoice"},{"type":"text","text":" method in the "},{"type":"codeVoice","code":"IdVerificationEventDelegate"}],"title":"IdVerificationError","type":"topic","url":"\/documentation\/idverification365id\/idverificationerror"},"doc://com.365id.sdk.idverification/documentation/IdVerification365id/IdVerificationError/serverError":{"identifier":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/IdVerificationError\/serverError","type":"topic","title":"IdVerificationError.serverError","role":"symbol","kind":"symbol","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"serverError"}],"abstract":[{"type":"text","text":"Happens when there is an error thrown by the 365id server"}],"url":"\/documentation\/idverification365id\/idverificationerror\/servererror"}}}